examples/c/Makefile.individual

```diff
@@ -0,0 +1,10 @@
+BINARY := $(notdir $(realpath .))
+TARGET := $(BINARY)/$(BINARY)
+
+.PHONY: all
+all:
+	@make --no-print-directory -C .. $(TARGET)
+
+.PHONY: clean
+clean:
+	rm $(BINARY)
```

Comment:
well this is not exactly cleaning all the relevant files, actually. $(OUTPUT)/$(BINARY) should be cleaned at least. We can add `make clean-$(BINARY)` targets for each known app, that would be nicer, probably.

Reply:
I think it would be better to add a `-f` flag to `rm`.

Reply:
Ok